paimon-spark/paimon-spark-common/src/main/scala/org/apache/paimon/spark/commands/MergeIntoPaimonTable.scala

```diff
@@ -58,6 +59,8 @@ case class MergeIntoPaimonTable(
 
   override val table: FileStoreTable = v2Table.getTable.asInstanceOf[FileStoreTable]
 
+  lazy val spark: SparkSession = SparkSession.active
```

Comment:
we should avoid to use `SparkSession.active`, in some cases, it will miss configs.